ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/ipv4-subnet/provision/cucushift-installer-rehearse-aws-ipi-ovn-ipv4-subnet-provision-chain.metadata.json

```
@@ -0,0 +1,21 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/ovn/ipv4-subnet/provision/cucushift-installer-rehearse-aws-ipi-ovn-ipv4-subnet-provision-chain.yaml",
```

To keep consistent with azure, suggest:
```suggestion
	"path": "cucushift/installer/rehearse/aws/ipi/sharednetwork/ovn/ipv4-subnet/provision/cucushift-installer-rehearse-aws-ipi-sharednetwork-ovn-ipv4-subnet-provision-chain.yaml",
```

---

I guess here is covered the scenario of configuring ipv4 subnet on normal cluster, as we discussed before in [slack thread](https://redhat-internal.slack.com/archives/CF8SMALS1/p1732678720048769?thread_ts=1732677849.030769&cid=CF8SMALS1).

---

I saw `aws-provision-vpc-shared` is called in the install chain, leave this to yunfei to confirm

---

The `byo-vpc` or `sharednetwork` are not called explicitly in AWS, e.g. `cucushift-installer-rehearse-aws-ipi-byo-route53`, it called `aws-provision-vpc-shared` implicitly, so following this rule, I did not add extra `byo-vpc` or `sharednetwork`, as `ipv4-subnet` must be under BYO-VPC.

---

> I guess here is covered the scenario of configuring ipv4 subnet on normal cluster, as we discussed before in [slack thread](https://redhat-internal.slack.com/archives/CF8SMALS1/p1732678720048769?thread_ts=1732677849.030769&cid=CF8SMALS1).

Ah, right, please ignore my above comment. 

AWS could be configured as a non-byo-vpc scenario for this case.

Let me update.

---

byo-vpc get passed https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/60071/rehearse-60071-periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-ovn-ipv4-subnet-arm-f14/1869968484734078976

---

> AWS could be configured as a non-byo-vpc scenario for this case.

If that, when we cover both, we still need to distinguish byo-vpc and non-byo-vpc for the case, right?

---

AWS covers only one configuration non-byo-vpc is enough I think, as Azure has covered the other configuration.
The new name like `cucushift-installer-rehearse-aws-ipi-non-byo-vpc-ovn-ipv4-subnet` or `cucushift-installer-rehearse-aws-ipi-non-sharednetwork-ovn-ipv4-subnet`?

---

If that, I prefer `cucushift-installer-rehearse-aws-ipi-ovn-ipv4-subnet`